### ISSUE TITLE
[2021/03/18]Machine Learning 輪講

### ISSUE THREAD

**Issue by hakubishin3** [wantedly/machine-learning-round-table#91]
## Why

Machine Learning 輪講は最新の技術や論文を追うことで、エンジニアが「技術で解決できること」のレベルをあげていくことを目的にした会です。

prev.

## What

話したいことがある人はここにコメントしましょう！
面白いものを見つけた時点でとりあえず話すという宣言だけでもしましょう！

**Comment by hakubishin3** [comment]
## なんか紹介する

**Comment by hiroto0227** [comment]
## Are Neural Rankers still Outperformed by Gradient Boosted Decision Trees?
- https://openreview.net/forum?id=Ut1vF_q_vC
- ICLR 2021 Google Research

### どんな論文か？
- Neural Networkではランキング学習(Learning To Rank(LTR))が、GBDTなどのTree系アルゴリズムよりも性能がよくないケースが多い。
- 原因として考えられるのは以下の三つである。
  - Neural Networkは入力にセンシティブなため、入力のスケールを合わせる必要がある。
  - ベンチマークに使用しているデータセットが比較的小規模なため、学習しきれていないか、overfitしている。
  - 一般の全結合層では情報が十分に伝わらないが、LTRのNNでは全結合層を使用していることが多い。
- それらに対応するためにNeural Networkの構造をランキング学習に最適化したところ、GBDTと同じかそれよりも良いスコアとなった。

まとめ: https://github.com/hiroto0227/paper/issues/3
